scalr/resource_scalr_workspace.go

```diff
@@ -159,6 +163,7 @@ func resourceTFEWorkspaceCreate(d *schema.ResourceData, meta interface{}) error
 			Identifier:        scalr.String(vcsRepo["identifier"].(string)),
 			IngressSubmodules: scalr.Bool(vcsRepo["ingress_submodules"].(bool)),
 			OAuthTokenID:      scalr.String(vcsRepo["oauth_token_id"].(string)),
+			Path:              scalr.String(vcsRepo["path"].(string)),
```

Comment:
FYI this will lead to `panic: interface conversion: interface {} is nil, not string` if used on a version of Scalr before this field was added. Need to think how to handle this.

Reply:
@lbn I forgot to add:

>  Provider requires scalr server at least 20200707 (mention that in the provider change log).

Maybe this will be enough not to handle unnecessary cases?